index.html

```diff
@@ -162,16 +167,16 @@ <h2 class="work-title">My Recent Works</h2>
         </svg>
       </div>
       <div class="projects">
-       <div class="projects-card">
+        <div class="projects-card">
           <div class="image-holder"></div>
           <div class="text-container">
-            <div class="project-header">Multi-Post Stories Gain+Glory</div>
-              <ul class="project-language">
-                <li class="language ruby">Ruby on rails</li>
-                <li class="language css">css</li>
-                <li class="language js">JavaScript</li>
-                <li class="language html">html</li>
-              </ul>
+            <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+            <ul class="project-language">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language css">css</li>
+              <li class="language js">JavaScript</li>
+              <li class="language html">html</li>
+            </ul>
             <div class="button">
               <button type="button">See Project</button>
             </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add the correct spacing between project cards to match the Figma design by fixing the overlapping issues of cards.

| Figma Design | Your DEsig |
| -------------------- | ---------------- |
| ![Screenshot from 2022-09-27 16-38-41](https://user-images.githubusercontent.com/83042742/192557028-e19e25b1-2c43-438b-9400-ecc2602e1e99.png) | ![Screenshot from 2022-09-27 16-36-03](https://user-images.githubusercontent.com/83042742/192557092-da2444f3-2a29-4c72-a8ec-ca22ed63e0ca.png) |